### ISSUE TITLE
Typo in example code for Body

### ISSUE THREAD

**Issue by torfsen** [OParl/spec#349]
The example JSON code for `Body` contains a typo:

```
...
    "location": {
        "id:": "https://oparl.example.org/location/0", 
        "type": "https://schema.oparl.org/1.0/Location", 
...
```
That should be `"id"` instead of `"id:"` (remove the colon).

**Comment by eFrane** [comment]
This relates to #350 and will be resolved accordingly as either fix to the existing or due to replacement with "real" examples.

**Comment by konstin** [comment]
Fixed without #350 for now.
